### DISCUSSION TITLE
Occasional Error: Failed to compile Typescript processor

### DISCUSSION THREAD

**Discussion by shiva-taramani** [artilleryio/artillery#3255] [Artillery]
Hi! When I run my test locally, I occasionally see the following error. I am using Playwright with Typescript.

```
assertion failed [block != nullptr]: BasicBlock requested for unrecognized address
(BuilderBase.h:550 block_for_offset)
     Error: Failed to compile Typescript processor
    The service was stopped
```

Any help would be much appreciated!

**Comment by shiva-taramani** [comment]
There is a possibility that this is because of my M1 mac. https://github.com/ethereum/solidity/issues/13523#issuecomment-1544599434

I will dig.

**Reply by hassy** [reply]
did you find out if it was a specific dependency that was causing it @sthavamani-ut?

**Reply by shiva-taramani** [reply]
Yes, I had to reinstall node. I have not seen any issues since I did that.

**Reply by hassy** [reply]
Ah okay, good to know, thanks!
